newton/tests/kamino/test_kamino_solver_kamino.py

```
@@ -1288,7 +1310,18 @@ def test_11_step_multiple_worlds_from_initial_state_without_contacts(self):
         msg.info(f"[samples]: [single]: [final]: w_i (shape={final_w_i.shape}):\n{final_w_i}\n")
         msg.info(f"[samples]: [single]: [final]: q_j (shape={final_q_j.shape}):\n{final_q_j}\n")
         msg.info(f"[samples]: [single]: [final]: dq_j (shape={final_dq_j.shape}):\n{final_dq_j}\n")
-        msg.info(f"[samples]: [single]: [final]: lambda_j (shape={final_lambda_j.shape}):\n{final_lambda_j}\n")
+        msg.info(
+            f"[samples]: [single]: [final]: lambda_kin_j (shape={final_lambda_kin_j.shape}):\n{final_lambda_kin_j}\n"
```

Ruff (0.16.1)

[error] Undefined name `final_lambda_kin_j`

(F821)